docs/en/ingest-management/tab-widgets/install-layout.asciidoc

```diff
@@ -46,6 +50,8 @@ Log files for {agent}footnote:lognumbering[]
 `C:\Program Files\Elastic\Agent\data\elastic-agent-*\logs\default\*-json.log*`::
 Log files for {beats} shippers
 
+You can install {agent} in a custom base path other than `C:\Program Files`.  When installing {agent} with the `./elastic-agent install`
```

Comment:
Fixing path separator and adding extension, since it's Windows. I had it wrong in the original PR (https://github.com/elastic/ingest-docs/pull/160) as well.  But what I have below is the correct invocation; see https://www.elastic.co/guide/en/fleet/current/install-standalone-elastic-agent.html.

```suggestion
You can install {agent} in a custom base path other than `C:\Program Files`.  When installing {agent} with the `.\elastic-agent.exe install`
```